README.md

```diff
@@ -77,7 +77,7 @@ Go to https://start.spring.io/ and initialize an Spring Boot app with `Web` and
 
 Customize pom to copy content from Frontend for serving it later with the embedded Tomcat:
 
-```
+```xml
```

Comment:
Wow @DamianFekete thanks for pointing that out to me - didn´t know that Markdown goody 😃